Networking/NetworkingTests/Remote/SitePluginsRemoteTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+@testable import Networking
+
+/// SitePluginsRemote Unit Tests
+///
+class SitePluginsRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    /// Dummy Site ID
+    ///
+    let sampleSiteID: Int64 = 1234
+
+    /// Repeat always!
+    ///
+    override func setUp() {
+        network.removeAllSimulatedResponses()
+    }
+
+    // MARK: - Load plugins tests
+
+    /// Verifies that loadPlugins properly parses the sample response.
+    ///
+    func testLoadPluginsProperlyReturnsPlugins() {
+        let remote = SitePluginsRemote(network: network)
+        let expectation = self.expectation(description: "Load site plugins")
```

Comment:
nit: also here we started to abandon the `expectation` and the `wait(for: [expectation], timeout: Constants.expectationTimeout)` methods, and we started to use this shape (the code below is not tested 🗡️ ):

```
// Given
        let remote = SitePluginsRemote(network: network)
        network.simulateResponse(requestUrlSuffix: "plugins", filename: "plugins")

        // When
        let result: Result<[SitePlugin], Error> = waitFor { promise in
            remote.loadPlugins(for: sampleSiteID) { result in
                promise(result)
            }
        }

        // Then
        let sitePlugins = try XCTUnwrap(result.get().plugins)
        let errors = try result.get().errors
        XCTAssertEqual(sitePlugins?.count, 5)
        XCTAssertNil(errors)
```

Reply:
Ended up doing more like this:

```
        // When
        let result: Result<[SitePlugin], Error> = waitFor { promise in
            remote.loadPlugins(for: self.sampleSiteID) { result in
                promise(result)
            }
        }

        // Then
        switch result {
        case .success(let plugins):
            XCTAssertEqual(plugins.count, 5)
        case .failure(let error):
            XCTAssertNil(error)
        }
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/8e320fa88e27122e36d57a3de6e396bc6ec0ce37